Lombiq.HelpfulLibraries.OrchardCore/Mvc/TypedRoute.cs

```diff
@@ -91,7 +91,9 @@ public string ToString(HttpContext httpContext)
     /// </summary>
     public override string ToString()
     {
-        var prefix = _isAdminLazy.Value ? "/Admin/" : "/";
+        var isAdminWithoutRoute = _isAdminLazy.Value && _action.GetCustomAttribute(typeof(RouteAttribute)) == null;
+
+        var prefix = isAdminWithoutRoute ? "/Admin/" : "/";
```

Comment:
I don't know the context but is it correct to use an hardcoded `/Admin` prefix, shouldn't it be retrieved from the current `AdminOptions.AdminUrlPrefix`?

Reply:
https://github.com/Lombiq/Helpful-Libraries/issues/209